python/tvm_ffi/cython/function.pxi

```diff
@@ -501,6 +501,8 @@ cdef int TVMFFIPyArgSetterDTypeFromNumpy_(
     out.v_dtype = NUMPY_DTYPE_TO_DTYPE[py_obj]
     return 0
 
+cdef _DISPATCH_TYPE_KEEP_ALIVE = set()
```

Comment:
I was worried if there're concurrent calls from multiple threads - in this case will this corrupt this global variable?

Reply:
great call out